src/Controllers/controller.py

```diff
@@ -26,7 +26,9 @@ def run(self):
         while not self._should_quit:
             try: 
                 data = self.parse(self._controller.request_update())
-                ret = self._controller.update(data)
+                ret = data[0](*data[1:])
+                if isinstance(ret, str):
+                    print(ret) 
```

Comment:
this doesn't work because it will print in gui mode too.

Reply:
which we were explicitly warned against at the end of last sprint. This is why it was originally separated.

Reply:
Added a check for CLI mode